### PR TITLE
Workflow for automated player version update PR creation

### PR DESCRIPTION
**Pull request by matamegger**
A first workflow to automate the version update via external triggers.

### CODE REVIEW

File: .github/workflows/create-sdk-update-pr.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Git User
+        run: |
+          git config --global user.name "Update Bot"
+          git config --global user.email "update-bot@bitmovin.com"
+
+      - name: Set Update Branch name
```
Comment (strangesource) [inline .github/workflows/create-sdk-update-pr.yml:32]: NIT: job names are not consistent with upper and lower case characters.

File: .github/workflows/create-sdk-update-pr.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        run: |
+          branch_name="update_${{ inputs.sdk_name}}_player_to_${{ inputs.version_number }}"
+          echo "branch_name=$branch_name" >> $GITHUB_OUTPUT
+
+      - name: Create Update Branch
+        run: |
+          git push origin --delete ${{ steps.branching.outputs.branch_name }} || true
+          git checkout -b ${{ steps.branching.outputs.branch_name }}
+
+      - name: Bump iOS SDK version
```
Comment (strangesource) [inline .github/workflows/create-sdk-update-pr.yml:43]: Nit, same goes for Android.
```suggestion
      - name: Bump iOS Player SDK version
```

File: .github/workflows/create-sdk-update-pr.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          git add RNBitmovinPlayer.podspec android/build.gradle
+          git commit -m "chore(${{ inputs.sdk_name }}): update ${{ inputs.sdk_name }} player version to ${{ inputs.version_number }}"
+          git push origin ${{ steps.branching.outputs.branch_name }}
+
+      - name: Create PR
+        run: |
+          gh pr create \
+          --base "${{ github.ref }}" \
+          --title "Update ${{ inputs.sdk_name }} player to ${{ inputs.version_number }}" \
+          --body "Automated ${{ inputs.sdk_name }} player version update to ${{ inputs.version_number }}"
```
Comment (strangesource) [inline .github/workflows/create-sdk-update-pr.yml:64]: Does the github CLI reuse the token from the environment?

**Review by strangesource — COMMENTED** [review]
Looking good in general, just some NIT.

File: .github/workflows/create-sdk-update-pr.yml (hunk 1 above)
Comment (matamegger) [reply .github/workflows/create-sdk-update-pr.yml]: https://github.com/bitmovin/bitmovin-player-react-native/pull/87/commits/ac102b1321b860928978cc50d7f4c3eeec03e766

File: .github/workflows/create-sdk-update-pr.yml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+      - name: Setup Git User
+        run: |
+          git config --global user.name "Update Bot"
+          git config --global user.email "update-bot@bitmovin.com"
+
+      - name: Set Update Branch name
+        id: branching
+        run: |
+          branch_name="update_${{ inputs.sdk_name}}_player_to_${{ inputs.version_number }}"
```
Comment (rolandkakonyi) [inline .github/workflows/create-sdk-update-pr.yml:35]: nit: Should we use `feature/` prefix?

**Review by rolandkakonyi — APPROVED** [review]
Nice job!

**Review by strangesource — APPROVED** [review]

File: .github/workflows/create-sdk-update-pr.yml (hunk 4 above)
Comment (matamegger) [reply .github/workflows/create-sdk-update-pr.yml:35]: We have no convention in the repository. I would prefer to not use it. IMO there is no gain.
(I have to touch the file again soon. If you have a strong opinion, we can still change this 🙂)

**Pull request merged**